### DISCUSSION TITLE
Create offer for an existing ASIN

### DISCUSSION THREAD

**Discussion by JesusR25** [amzn/selling-partner-api-models#4433] [Q&A] [answered]
Hello, let's say I have a listing that I removed from an ASIN, but now I want to list it again (with a different SKU). What would this process look like? I can't find much information on linking it to an existing ASIN. You could say I already have all the necessary information to make the request, but I’m not very familiar with the flow. As I mentioned, I don’t see clear documentation for this case.

Thank you very much for reading!

**Comment by ritikacd** [comment, marked as answer]
Hi @JesusR25 

You can create an offer to list an item for ASIN that already exists in the Amazon catalog, here's the [documentation](https://developer-docs.amazon.com/sp-api/docs/building-listings-management-workflows-guide#list-an-offer-for-an-item-that-already-exists-in-the-amazon-catalog) for your use case.  The documentation also provides flow-diagram to understand how data flows through distinct steps. 

Once you successfully create an offer, you can get the details of the new listing item using [getListingsItem](https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference#get-listings2021-08-01itemsselleridsku) operation by passing the seller SKU. 

Thanks,
Ritika
Selling Partner API Services

**Reply by JesusR25** [reply]
@ritikacd Thanks for your answer, I will check the documentation! 😃
